dependencies.gradle

```diff
@@ -15,6 +15,7 @@ def gradle = "7.1.3"
 def kotlin = "1.6.21"
 def kotlinCoroutines = "1.6.4"
 def dagger = "2.42"
+def appDistribution = "16.0.0-beta03"
```

Comment:
Is there any stable version instead?

Reply:
I was just following the doc: https://firebase.google.com/docs/app-distribution/set-up-alerts?platform=android

<img width="924" alt="image" src="https://user-images.githubusercontent.com/3940906/180729103-1dc3af9e-0bbe-40d2-a504-b5081b9399c0.png">

I guess a stable version will be release someday and Dependabot will do the rest.